kornia/morphology/morphology.py

```diff
@@ -0,0 +1,128 @@
+# Libraries
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import kornia.morphology as m
```

Comment:
to avoid confusions and potential issues with debuggers, use a better module name e.g `morph`